mypy/main.py

```diff
@@ -28,6 +28,21 @@ class InvalidPackageName(Exception):
     """Exception indicating that a package name was invalid."""
 
 
+orig_stat = os.stat
+
+
+def stat_proxy(path: str) -> os.stat_result:
```

Comment:
Hm, we probably don't need this function. (It was an experiment to count stat() calls, and while the numbers are useful, the implementation feels pretty wrong.)